src/index.html

```diff
@@ -1,12 +1,388 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div>
+        <a href="#home">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div>
```

Comment:
do you need div here? 
As i can see this div have size 40*43.6, but logo is 40*40.
Thats why logo is not at the center of header